1001-2000/LIT1899Martyr.xml

```diff
@@ -44,7 +36,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <langUsage><language ident="en">English</language><language ident="ar">Arabic</language><language ident="gez">Gǝʿǝz</language></langUsage>
+         <abstract>
+            <p>Unspecified text of Martyrium / Gadl of St Mercurios.</p>
```

Comment:
I think it's probably better to use "Mercurius", which seems to be the standard form in English (https://en.wikipedia.org/wiki/Saint_Mercurius).

This applies also elsewhere in these files

Reply:
> I think it's probably better to use "Mercurius", which seems to be the standard form in English (https://en.wikipedia.org/wiki/Saint_Mercurius).
> 
> This applies also elsewhere in these files

it was created before me as Mercurios, but you are right, I will change it!